packages/babel-node/README.md

```diff
@@ -0,0 +1,19 @@
+# babel-node
+
+> Babel node
+
+babel comes with a second CLI which works exactly the same as Node.js’s CLI, only it will compile ES6 code before running it.
```

Comment:
Since it's now separate, thoughts on rewording to something like:

```
> A Babel-powered Node.js CLI

babel-node is a CLI that works exactly the same as the Node.js 
CLI, with the added benefit of compiling with Babel presets and 
plugins before running it.
```

?